strings/remove_duplicates.py

```diff
@@ -0,0 +1,22 @@
+# Created by sarathkaul on 14/11/19
+
+from ordered_set import OrderedSet
+
+
+def remove_duplicates(sentence: str) -> str:
+    """
+    Reomove duplicates from sentence
+    >>> remove_duplicates("Python is great and Java is also great")
+    'Python is great and Java also'
+    """
+    sen_list = sentence.split(" ")
+    check = OrderedSet()
+
+    for a_word in sen_list:
+        check.add(a_word)
+
+    return " ".join(check)
```

Comment:
return " ".join(sorted(check))  # and lose the dependency.

Reply:
That seems correct